src/clj/puppetlabs/http/client/async.clj

```diff
@@ -173,11 +173,12 @@
     metric-registry :- (schema/maybe MetricRegistry)
     metric-namespace :- (schema/maybe schema/Str)]
    (let [result (promise)
-         defaults {:headers {}
-                   :body nil
+         defaults {:body nil
                    :decompress-body true
                    :as :stream}
-         opts (merge defaults opts)
+         headers (merge {"Accept-Language" (str (i18n/user-locale))} (:headers opts))
```

Comment:
@lutter Is there a more appropriate way to format the locale for the header here?